### ISSUE TITLE
Bug report - Files aren't saved anymore (iOS)

### ISSUE THREAD

**Issue by LucaAtWork** [localsend/localsend#2025]
### Describe the bug

In the past I was able to access my transfered files on my iOS device by using File app and opening LocalSend directory. 
I've reinstalled LocalSend recently and today all transfered files from my Linux desktop to my iPhone aren't saved to that directory.
I've disabled the in-app transfers "history" as I don't need it.
So, where are my transfered files located?

### To reproduce

1. Transfer any file from Desktop to iOS
2. Complete the transfer
3. Where is my file located?? No files in LocalSend directory.

### Expected behavior

In the past I was able to access all my transfered files from the LocalSend directory on the root of the File app.

### Desktop (please complete the following information)

- OS: ArchLinux
- LocalSend: 1.16.1

### Smartphone (please complete the following information)

- OS: iOS 18.1
- LocalSend: 1.16.2

**Comment by LucaAtWork** [comment]
@Tienisto yeah, but I'm refering to all the "new" transfers. At the moment if you trasfer a file to an iOS device, apparently it will NOT be saved anywhere (if you disable history).

**Comment by milicodex** [comment]
> @Tienisto yeah, but I'm refering to all the "new" transfers. At the moment if you trasfer a file to an iOS device, apparently it will NOT be saved anywhere (if you disable history).

On iOS when you open Settings and scroll down to the Localsend app there are a few extra settings there, one of them is Document Storage with a few options. What folder this setting says?

**Comment by LucaAtWork** [comment]
> > @Tienisto yeah, but I'm refering to all the "new" transfers. At the moment if you trasfer a file to an iOS device, apparently it will NOT be saved anywhere (if you disable history).
> 
> On iOS when you open Settings and scroll down to the Localsend app there are a few extra settings there, one of them is Document Storage with a few options. What folder this setting says?

Hi @milicodex , it says "iPhone".

**Comment by milicodex** [comment]
> > > @Tienisto yeah, but I'm refering to all the "new" transfers. At the moment if you trasfer a file to an iOS device, apparently it will NOT be saved anywhere (if you disable history).
> > 
> > 
> > On iOS when you open Settings and scroll down to the Localsend app there are a few extra settings there, one of them is Document Storage with a few options. What folder this setting says?
> 
> Hi @milicodex , it says "iPhone".

Mine says On My iPhone and when opening the iOS Files app I can see diffrent Location like "iCloud Drive" "On My iPhone" "Recently Deleted" etc. when I tap On My iPhone there is a LocalSend folder where the transfered files can be found.
Does this help?

**Comment by LucaAtWork** [comment]
> > > > @Tienisto yeah, but I'm refering to all the "new" transfers. At the moment if you trasfer a file to an iOS device, apparently it will NOT be saved anywhere (if you disable history).
> > > 
> > > 
> > > On iOS when you open Settings and scroll down to the Localsend app there are a few extra settings there, one of them is Document Storage with a few options. What folder this setting says?
> > 
> > 
> > Hi @milicodex , it says "iPhone".
> 
> Mine says On My iPhone (I have the option to change to "iCloud Drive") and when opening the iOS Files app I can see diffrent Locations like "iCloud Drive" "On My iPhone" "Recently Deleted" etc. When I tap On My iPhone there is a LocalSend folder where the transfered files can be found. Does this help?

No :( I have the "LocalSend" directory on the root of the File app, but it is empty.
In the past, whenever I completed a transfer, the transferred files were saved there.
I also checked cloud storage: there is no "LocalSend" directory there.

**Comment by Tienisto** [comment]
What did you transferred? By default, LocalSend puts media files into the photos app.

**Comment by LucaAtWork** [comment]
> What did you transferred? By default, LocalSend puts media files into the photos app.

I've disabled that behavior in LocalSend settings, so I expect that photos will be stored in LocalSend folder too.
Anyway, I'm trasfering mainly PDFs and occasionally some .zip files.
